tool/src/webapp/js/forum-summary.js

```diff
@@ -0,0 +1,59 @@
+var itemsToShow;
+$(function() {
+	//escape markup
+	String.prototype.escapeHTML = function(){
```

Comment:
Messing with String.prototype can be risky as another library can end up replacing your function with something different.